### PR TITLE
Inspec additional Tests

### PR DESCRIPTION
**Pull request by admiralAwkbar**
- Adding Inspec tests to the test suite

### CODE REVIEW

**Comment by ferrarimarco** [comment]
@admiralAwkbar I'd try with the [InSpec shell](https://docs.chef.io/inspec/cli/#shell), and run the same check there and see if it keeps failing

**Comment by ferrarimarco** [comment]
Plus, there are more leftovers to remove :)

See the [TEST_CASE_RUN variable](https://github.com/github/super-linter/search?q=TEST_CASE_RUN), and where it's called (including [a workflow step](https://github.com/github/super-linter/blob/master/.github/workflows/deploy-DEV.yml#L134))

**Comment by admiralAwkbar** [comment]
added #1509 to track the issue with those npm packages

File: Dockerfile [hunk 1]
```diff
@@ -214,7 +214,7 @@ COPY --from=dockerfile-lint /bin/hadolint /usr/bin/hadolint
 #################
 # Install lintr #
 #################
-COPY --from=lintr-lib /usr/lib/R/library/ /home/r-library
+COPY --from=lintr-lib /usr/lib/R/library/ /usr/lib/r-library
```
Comment (ferrarimarco) [inline Dockerfile:217]: I'd add a test for this (and similar directives). See https://docs.chef.io/inspec/resources/file/

**Review by ferrarimarco — CHANGES_REQUESTED** [review]

**Pull request merged**